src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,190 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "insert into cars (model, manufacturer_id) values (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement
+                        = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2, car.getManufacturer().getId());
+            statement.executeUpdate();
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                car.setId(generatedKeys.getLong(1));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not create a car! ", e);
+        }
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Car get(Long id) {
+        String query = "select c.id as car_id, c.model, m.id as manufacturer_id, m.name, "
+                + "m.country from cars c join manufacturers m on c.manufacturer_id = m.id "
+                + "where c.id = ? and c.is_deleted = false and m.is_deleted = false;";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                car = parseCarWithManufacturer(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not get a car! ", e);
+        }
+        if (car != null) {
+            car.setDrivers(getCarDrivers(id));
+        }
+        return car;
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String query = "select * from cars where is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            List<Car> cars = new ArrayList<>();
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(get(resultSet.getLong("id")));
+            }

```

Comment:
That is not how method should work. You make 1 sql query to select all cars and then make 1 additional query to DB for each car found.
Please make 1 proper query to get all required fields and reuse your parseCarWithManufacturer method in cycle here to retrieve all models from 1 result set